composer.json

```diff
@@ -22,17 +22,26 @@
     },
     "autoload": {
         "psr-4" : {
-            "chobie\\" : "src"
+            "chobie\\" : "src/"
         }
     },
     "autoload-dev": {
         "psr-4": {
-            "Tests\\chobie\\": "tests"
+            "Tests\\chobie\\": "tests/"
         }
     },
+	"scripts": {
+		"stan": "phpstan analyse",
+		"stan-setup": "cp composer.json composer.backup && composer require --dev phpstan/phpstan:^1.0.0 && mv composer.backup composer.json"
+	},
```

Comment:
Please remove. Instead, do this inside a build workflow.

Theoretically, the `phpstan analyse` command should fail, because PHPStan is installed locally and therefore can only be accessed via `vendor/bin/phpstan`.